release/create-release-candidate-branch.sh

```diff
@@ -229,31 +278,48 @@ parse_inputs() {
 	echo "Settings: ${RELEASE_BRANCH}: Push: $PUSH: Cleanup: $CLEANUP"
 }
 
+check_dependencies() {
+	# test required dependencies:
+	git config --get user.name
+	# check gh authentication: if this fails you will need to e.g. gh auth login
+	gh auth status
+	yq --version
+	python --version
+	cargo --version
+	cargo set-version --version
+	# TODO: jinja2-cli including pyyaml package?
```

Comment:
note: We should check that jinja2 exists because the script will exit in a potentially inconsistent state.

Reply:
See e3f1764